src/components/DappCard.js

```diff
@@ -48,7 +48,8 @@ export default function DappCard({ dapp, actions = true }) {
           {dapp.addedOn && (
             <p className="text-palette-300 text-xs flex items-center flex-row truncate">
               <ClockIcon className="w-4 inline-flex mr-1 flex-shrink-0" />{" "}
-              {dapp.skylinks.length > 1 ? "updated" : "installed"} <RelativeDate date={dapp.addedOn} />
+              {dapp.skylinks.length > 1 ? "updated" : "installed"}
+              <RelativeDate date={dapp.skylinks[dapp.skylinks.length - 1].addedOn} />
```

Comment:
You're missing a space between the label and the relative time now.

```suggestion
              {dapp.skylinks.length > 1 ? "updated" : "installed"}{" "}
              <RelativeDate date={dapp.skylinks[dapp.skylinks.length - 1].addedOn} />
```

<img width="445" alt="Screenshot 2021-12-20 at 17 45 53" src="https://user-images.githubusercontent.com/3755029/146802702-289f5c1a-6c45-4c85-9fbf-860cab052d12.png">
<img width="441" alt="Screenshot 2021-12-20 at 17 45 29" src="https://user-images.githubusercontent.com/3755029/146802704-ad8763d7-8bf4-4301-8308-cb040de791dd.png">

Reply:
Oh thats what that char was for 😅 I though that I added that by accident